e2e/test/scenarios/question/reproductions/22230-query-builder-aggregation-filtering.cy.spec.js

```diff
@@ -1,38 +0,0 @@
-import {
```

Comment:
This is a duplicate of `e2e/test/scenarios/filters/reproductions/22230-filter-on-aggregation-max-of.cy.spec.js` (the file above)